packages/dataviews/src/dataviews-layouts/grid/index.tsx

```diff
@@ -56,6 +57,14 @@ interface GridItemProps< Item > {
 	hasBulkActions: boolean;
 }
 
+function chunk( array: any, size: any ) {
```

Comment:
Given that we know what we chunk, and `size` is definitely a number, can we avoid using `any` types here?